EFCore/MAUI/Readme.md

```diff
@@ -0,0 +1,856 @@
+
+  Alternatively, if you only want to run the example project or use the project as a boilerplate for your application, you can manually register your [NuGet feed URL](https://docs.devexpress.com/GeneralInformation/116042/installation/install-devexpress-controls-using-nuget-packages/obtain-your-nuget-feed-url) in Visual Studio as described in the following article: [Setup Visual Studio's NuGet Package Manager](https://docs.devexpress.com/GeneralInformation/116698/installation/install-devexpress-controls-using-nuget-packages/setup-visual-studios-nuget-package-manager)
+  
+  > **NOTE** 
+  >
+  > If you have a pre-release version of our components, for example, provided with the hotfix, you also have a pre-release version of NuGet packages. These packages will not be restored automatically and you need to update them manually as described in the [Updating Packages](https://docs.devexpress.com/GeneralInformation/118420/Installation/Install-DevExpress-Controls-Using-NuGet-Packages/Updating-Packages) article using the [Include prerelease](https://docs.microsoft.com/en-us/nuget/create-packages/prerelease-packages#installing-and-updating-pre-release-packages) option.
+
+## Table of Contents
+
+[**Build and Test a Basic Web API Service App (Backend)**](#build-and-test-a-basic-web-api-service-app-backend)
+
+- [Create a Web API project using the XAF Solution Wizard](#create-a-web-api-project-using-the-xaf-solution-wizard)
+- [Declare a data model](#declare-a-data-model)
+- [Set up a development database connection](#set-up-a-development-database-connection)
+- [Generate initial data](#generate-initial-data)
+- [Use Swagger UI to test the Web API service](#use-swagger-ui-to-test-the-web-api-service)
+
+[**Build and test a basic .NET MAUI App (Frontend)**](#build-and-test-a-basic-net-maui-app-frontend)
+
+- [Create a .NET MAUI project](#create-a-net-maui-project)
+- [Modify platform specific settings](#modify-the-platform-specific-settings)
+- [Define the application's data model](#define-the-applications-data-model)
+- [Implement and test first views: "Login" and "Item List"](#implement-and-test-first-views-login-and-item-list)
+
+[**Extend backend and frontend apps**](#extend-backend-and-frontend-apps)
+
+- [Add an "Item Details" View](#add-an-item-details-view)
+- [Add a "New Item" entry form](#add-a-new-item-entry-form)
+- [Allow users to archive records](#allow-users-to-archive-records)
+- [Add reporting functionality](#add-reporting-functionality)
+
+## Build and Test a Basic Web API Service App (Backend)
+
+### Create a Web API project using the XAF Solution Wizard
+
+1. Start the wizard and select a Web API only project.
+  ![](../../images/MAUI/SolutionWizardWebAPI.png)
```

Comment:
It would be great to add borders to Wizard screenshots. Sometimes it looks like the top part of the screenshot belong to the text above and then there's another separate screenshot that's related to text below. There's so much white space in the middle.

Reply:
I edited the images to make them easier on the eyes.